.gitattributes

```diff
@@ -7,8 +7,18 @@
 *.png binary
 
 # Remove files for archives generated using `git archive`
-phpunit.xml.dist export-ignore
-.travis.yml export-ignore
-.gitignore export-ignore
-.gitattributes export-ignore
 .editorconfig export-ignore
+.env export-ignore
+.gitattributes export-ignore
+.gitignore export-ignore
+.travis.yml export-ignore
+phpcs.xml export-ignore
+phpstan.neon export-ignore
+phpunit.xml.dist export-ignore
+compose.yaml export-ignore
+Makefile export-ignore
+
+# Remove directories for archives
+.docker export-ignore
+.github export-ignore
+tests export-ignore
```

Comment:
we might want to keep tests, as they usually show usage.

Reply:
Hmmm I think it should be better that the README has a high quality instead to keep a unnecessary tests directory. It blows up the archive without any usage and increase the network traffic. :)

The usage should be available in README or the code should instruct the user.

What do you think?

Reply:
Fair enough, but best to add trailing slash / so it is clear those are directories, not files.